playbooks/aws/openshift-minion/launch.yml

```diff
@@ -65,5 +70,11 @@
       delay: 10
       with_items: ec2.instances
 
-# Apply the configs, seprate so that just the configs can be run by themselves
-- include: config.yml
+- name: "Configure instances"
```

Comment:
When looking at the [`create_cluster` function of `cluster.sh`](https://github.com/openshift/openshift-online-ansible/blob/master/cluster.sh#L32), we see that the create method:
* calls `cloud.rb … launch` which triggers the execution of the `launch.yml` playbook and then,
* calls `update_cluster` which calls `clourd.rb … config` which triggers the execution of the `config.yml` playbook.

So, with that script, `./cluster.sh create <env>` runs the `config.yml` playbook twice:
* Once because it is included from `launch.yml` and
* Once directly from `update_cluster`.